pkg/ottl/e2e/e2e_test.go

```diff
@@ -822,6 +822,18 @@ func Test_e2e_ottl_features(t *testing.T) {
 			statement: `set(attributes["test"], attributes["metadata"]["uid"])`,
 			want:      func(_ ottllog.TransformContext) {},
 		},
+		{
+			name:      "map value",
```

Comment:
Could you add a test that validates map literals work with functions that use `PMapGetter`?

Reply:
sure, will add a test case for that